index.html

```diff
@@ -6537,92 +6550,133 @@ <h2 id="instantiation-of-AudioWorkletNode-and-AudioWorkletProcessor">
+            </li>
+            <li>Return <var>node</var>.
+            </li>
+          </ol>
+          <p>
+            In order to process a control message for the construction of an
+            <a>AudioWorkletProcessor</a>, given a string <var>nodeName</var>, a
+            serialization record <var>processorPortSerialization</var>, and an
+            <a>AudioWorkletNode</a> <var>node</var>, perform the following
+            steps on the <a>rendering thread</a>. If any of these steps throws
+            an exception (either explicitly or implicitly), abort the rest of
+            steps and queue a task on the control thread to fire <a href=
+            "#widl-AudioWorkletNode-onprocessorstatechange"><code>processorstatechange</code></a>
+            event to <var>node</var>.
+          </p>
+          <ol>
+            <li>Let <var>processorPort</var> be <a href=
+            "https://html.spec.whatwg.org/multipage/infrastructure.html#structureddeserializewithtransfer">
+              StructuredDeserializeWithTransfer</a>(<var>processorPortSerialization</var>,
+              the current Realm).
+            </li>
+            <li>Let <var>processorConstructor</var> be the result of looking up
+            <var>nodeName</var> on the <a>AudioWorkletGlobalScope</a>'s <a>node
+            name to processor definition map</a>.
+            </li>
+            <li>If <var>processorConstructor</var> is <code>undefined</code>,
+            throw a <code>NotSupportedError</code> DOMException.
+            </li>
+            <li>Let <var>processor</var> be the result of
+            Construct(<a>processorConstructor</a>).
+            </li>
+            <li>If <var>processor</var> does not implement the
+            <a>AudioWorkletProcessor</a> interface, throw an
+            <code>"InvalidStateError"</code> DOMException. NOTE: this can occur
+            if the author-supplied constructor uses JavaScript's
+            return-override feature, or does not properly call
+            <code>super()</code>.
+            </li>
+            <li>Set <var>processor</var>'s <a href=
+            "#widl-AudioWorkletProcessor-port">port</a> to
+            <var>processorPort</var>.
             </li>
-            <li>Set <a>node reference</a> in <var>processorInstance</var> to
-            <var>this</var>.
+            <li>Set <var>processor</var>'s <a>node reference</a> to
+            <var>node</var>.
             </li>
-            <li>Set <a>processor reference</a> in <var>this</var> to
-            <var>processorInstance</var>.
+            <li>Set <var>node</var>'s <a>processor reference</a> to
+            <var>processor</var>.
             </li>
-            <li>Return <var>this</var> as a result of the construction.
+            <li>Queue a task the <a>control thread</a> to fire
+            <code>running</code> event to the associated
+            <a>AudioWorkletNode</a>.
```

Comment:
Also, should the communication of the parameter descriptors list to the control thread be mentioned here?